app/models/message.rb

```diff
@@ -27,4 +28,20 @@ def format_phone_number
 
     self.phone = self.phone.gsub(/[^0-9]/, '')
   end
+
+  RELEVANT_TERMS = %w[
+    delegate
+    bernie
+    sanders
+    hillary
+    clinton
+    hrc
+  ]
+
+  def contents_is_relevant
+    unless RELEVANT_TERMS.any? { |term| contents.include?(term) }
```

Comment:
:rotating_light: This will only pass if people have bad grammar. Needs to be case-insensitive.